.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,22 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          runtime-cgroups: /system.slice/crio.service
       ---
-      kind: KubeletConfiguration
       apiVersion: kubelet.config.k8s.io/v1beta1
-      cgroupDriver: cgroupfs
+      kind: KubeletConfiguration
+      cgroupDriver: systemd
+      cgroupRoot: /
+      systemCgroups: /system.slice
+      kubeletCgroups: /system.slice/kubelet.service
+      enforceNodeAllocatable:
+      - pods
+      - system-reserved
+      systemReservedCgroup: /system.slice
+      kubeReservedCgroup: /system.slice/kubelet.service
```

Comment:
EKS isolates kubelet from system, any reason why it's different here?
```
  "systemReservedCgroup": "/system",
  "kubeReservedCgroup": "/runtime"
```

Reply:
my uderstanding is that this describes where the cgroup for the systemd unit is, not where you want it to be

Reply:
i can't tell from skimming through the code, but their example in CLI parameter description would suggest the opposite. I guess it's fine to keep them in the same slice.